dataset.py

```diff
@@ -128,6 +128,20 @@ def pull_item(self, index):
 
         # 2. build list of annotation
         anno_list = []
+        if not dataset_item.attributes['detection']:
+            # NOTE: add annotation as background ( no object )
+            # label_id: -1 will be incremented and used for background
+            dataset_item.attributes['detection'] = [
```

Comment:
add an background annotation to prevent `anno_list` to be empty.

Reply:
> NOTE: add annotation as background ( no object )

Is this an official spec of "object detection" on Annotation tool?

Reply:
I think `background` should be skipped because there is nothing to be processed. We can detect `background` by heuristic like you did here.

Reply:
> Is this an official spec of "object detection" on Annotation tool?

yes, according to annotation dev team, some developers ask this spec and they developed this way.
and they told me that images without bounding boxes should be included in training data because it tells model that "no object in this image".

Reply:
> because it tells model that "no object in this image".

👀 
I don't think so. lol

Reply:
I'm not an expert in this field `¯\_(ツ)_/¯`

Reply:
lol